Spark_op/CP/Config/config_file.py

```diff
@@ -0,0 +1,15 @@
+from configparser import ConfigParser
+
+from Spark_op.CP.PathHandler.Path import ROOT_PATH_CREDS_SPARK
+
+config = ConfigParser()
+config.read(ROOT_PATH_CREDS_SPARK)
+
+
+def get_config_URL():
+    return f'jdbc:mysql://{config.get("MySQL", "user")}:{config.get("MySQL", "password")}@localhost:{config.get("MySQL", "port")}/{config.get("MySQL", "database")}'

```

Comment:
http://pymotw.com/2/ConfigParser/#combining-values-with-interpolation

alternatively, you can read the URI directly from the .ini file